Removing duplicated @counter-style-tests

vitorroriz opened the pull request:
#### 09ecf54321a567dd478a89b8e3873a4e3c5b1c7e
<pre>
Removing duplicated @counter-style-tests
<a href="https://bugs.webkit.org/show_bug.cgi?id=254729">https://bugs.webkit.org/show_bug.cgi?id=254729</a>
rdar://107410996

Reviewed by Tim Nguyen.

The modified tests/expected files are duplicated since WPT moved them
to their proper counter-style-at-rule directory.

They are already existent for WebKit at:
LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-at-rule/

* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-additive-symbols-syntax-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-additive-symbols-syntax.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-fallback-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-fallback.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-name-syntax-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-name-syntax.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-negative-syntax-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-negative-syntax.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-pad-syntax-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-pad-syntax.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-prefix-suffix-syntax-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-prefix-suffix-syntax.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-range-syntax-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-range-syntax.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-speak-as-syntax-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-speak-as-syntax.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-symbols-syntax-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-symbols-syntax.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-system-syntax-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-system-syntax.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/263102@main">https://commits.webkit.org/263102@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/98cb8d5c5b718cd90e667a137dfc2f824d61f6a5

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/3620 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/3661 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/3810 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/5046 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/32/builds/3897 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/3761 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/3709 "Built successfully") | [❌ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/3142 "2 flakes 103 failures") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/3664 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/15/builds/3905 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/3268 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/4869 "Built successfully") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/39/builds/1411 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/3238 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/3295 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/3213 "Passed tests") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/3297 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/4635 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/3673 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/17/builds/2972 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/3207 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/3238 "Passed tests") | | 
| [  ~~🛠 🧪 merge~~](https://ews-build.webkit.org/#/builders/19/builds/876 "The change is no longer eligible for processing.") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/3239 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/416 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/3486 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
We should just resync the css/css-counter-styles/ directory at some point

vitorroriz committed:
Removing duplicated @counter-style-tests
https://bugs.webkit.org/show_bug.cgi?id=254729
rdar://107410996

Reviewed by Tim Nguyen.

The modified tests/expected files are duplicated since WPT moved them
to their proper counter-style-at-rule directory.

They are already existent for WebKit at:
LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-at-rule/

* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-additive-symbols-syntax-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-additive-symbols-syntax.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-fallback-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-fallback.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-name-syntax-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-name-syntax.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-negative-syntax-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-negative-syntax.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-pad-syntax-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-pad-syntax.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-prefix-suffix-syntax-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-prefix-suffix-syntax.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-range-syntax-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-range-syntax.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-speak-as-syntax-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-speak-as-syntax.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-symbols-syntax-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-symbols-syntax.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-system-syntax-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-system-syntax.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/w3c-import.log:

Canonical link: https://commits.webkit.org/263102@main
